batch_job/src/work_queue_factory.c

```diff
@@ -1036,14 +1036,13 @@ int main(int argc, char *argv[])
 	}
 	else {
 		if((argc - optind) != 2) {
+			fprintf(stderr,"work_queue_factory: You must either give a project name with the -M option or master-name option with a configuration file, or give the master's host and port.\n");
```

Comment:
Sure, that makes sense. Would you rather I take that whole if statement and make it a separate case (if project_regex...else if argc - optind != 2...else...) or move the fprintf to a separate case? Is there a style preference?